mod/deckorate_attribution/set/type/reference.rb

```diff
@@ -87,7 +87,7 @@ def new_form_opts
   def edit_fields
     [
       :subject,
-      [:adaptation, title: "Adaptation"],
+      [:adaptation, title: "Adaptation <span style='color: red;'>*</span>"],
```

Comment:
I don't want to hard-code a narrow solution like this, for many reasons. One of them is that the edit interface is not the only place that uses this title.